spec/rails_helper.rb

```diff
@@ -30,7 +30,7 @@
   puts e.to_s.strip
   exit 1
 end
-Rails.logger = Logger.new(STDOUT)
+Rails.logger = Logger.new($stdout)
```

Comment:
このへんよく知らないので
どういう意図で書き換えたのか説明してもらってもいい？
あるいは、解説のリンクとかあればそれでもいいです。

Reply:
こちらはどういう状況ですか？

Reply:
変更前はRubocopを走らせると、このようなエラーが出ました。`Style/GlobalStdStream: Use $stdout instead of STDOUT.`

`(STDOUT)`から`($stdout)`に変更したのは、Rubocopのエラーを解消するためでした。（ドキュメントや参照した記事はほぼ`STDOUT`を使っています。）

Rubocopの警告に必ずしも則る必要ってないのでしょうか？

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GlobalStdStream
https://stackoverflow.com/questions/6671716/difference-between-stdout-and-stdout-in-ruby

Reply:
> (STDOUT)から($stdout)に変更したのは、Rubocopのエラーを解消するためでした。

なるほどですね。理解しました！
なんで変更したのか意図が分からなかったので質問しました。

> Rubocopの警告に必ずしも則る必要ってないのでしょうか？

警告には従ったほうがいいです。というよりは、
自分がプログラミングをしていく中でやりやすいよう、また分かりやすくキレイにするため設定するものなので、
どちらが正しいというわけではないので、自分にとってこれは不要だと思ったものは設定から変えたほうがいいです。
ただ、そういうルールが設けられているという事は何かしらの問題があるからなので、
できるだけ設定をしてそれに準拠したほうがいいです。

Reply:
なるほど、Rubocopの設定や効率性を検証しながら、設定したルールで警告０を目指す感じなのですね！
承知しました！